multiplatform-markdown-renderer/src/commonMain/kotlin/com/mikepenz/markdown/compose/elements/MarkdownText.kt

```diff
@@ -119,12 +119,14 @@ fun MarkdownText(
     val layoutResult: MutableState<TextLayoutResult?> = remember { mutableStateOf(null) }
     val imageState = rememberMarkdownImageState()
 
-    val placeholderState by derivedStateOf {
-        transformer.placeholderConfig(
-            imageState.density,
-            imageState.containerSize,
-            imageState.intrinsicImageSize
-        )
+    val placeholderState by remember {
```

Comment:
I think we should use the `imageState` here as key

Reply:
Indeed, rememberMarkdownImageState may return a new object,
 If LocalDensity changes